test/models/game_test.rb

```diff
@@ -1,7 +1,33 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @g = Game.create(name: "New Game", white_player_id: 1, black_player_id: 1)
+  end
+
+  test "board populated with pieces" do
+    expected = 32
+    actual = @g.pieces.count
+    assert_equal expected, actual
+  end
+
```

Comment:
Add two tests, similar to the one above, that tests for the correct count of pieces for both black and white